src/python/python.cpp

```diff
@@ -552,4 +552,21 @@ PYBIND11_MODULE(onnxruntime_genai, m) {
 
   m.def("set_current_gpu_device_id", [](int device_id) { Ort::SetCurrentGpuDeviceId(device_id); });
   m.def("get_current_gpu_device_id", []() { return Ort::GetCurrentGpuDeviceId(); });
-}
+
+  m.def("register_execution_provider_library", [](const std::string& provider_name, const std::wstring& path_wstr) {
+    // TODO: input python string to std::wstring conversion performed by pybind11 may not be working properly here: wrong string if including special characters
```

Comment:
Does ort also currently only allow this api to run on windows (and not on other platforms)?